torch/distributed/elastic/rendezvous/dynamic_rendezvous.py

```diff
@@ -86,14 +86,17 @@ class RendezvousTimeout:
 
     Args:
         join:
-            The total time within which the rendezvous is expected to complete.
+            The time within which the rendezvous is expected to complete.
```

Comment:
For the sake of uniformity, I think we need to pull `RendezvousTimeout` and `RendezvousParameter` to `api.py` and use these structs in etcd_rendezvous or just follow etcd_rendezvous' way of constructing the rdzv object by just passing all the required ctor-args flat. Docs will be duplicated as the verbiage is the same between the table in etcd rendezvous and this.

https://github.com/pytorch/pytorch/blob/master/torch/distributed/elastic/rendezvous/etcd_rendezvous.py#L203-L211

Reply:
I think the ideal solution would be to deprecate/remove the existing `EtcdRendezvousHandler` and to migrate to `DynamicRendezvousHandler` as it already supports both etcd and c10d. In such case we can also move `RendezvousBackend` and `RendezvousTimeout` to `api.py`.

If we all agree on its removal, I can have a follow-up PR after we merge `DynamicRendezvousHandler`.